application/F3DOptionsParser.cxx

```diff
@@ -274,6 +274,8 @@ void ConfigurationOptions::GetOptions(F3DAppOptions& appOptions, f3d::options& o
   std::vector<std::string>& inputs, std::string filePathForConfigBlock, bool allOptionsInitialized,
   bool parseCommandLine)
 {
+  inputs.clear(); /* that would not be necessary if this function wasn't called multiple times */
```

Comment:
It is needed to call it multipe time, maybe better to say:

"Required because this function is called multiple times"

Reply:
> It is needed to call it multipe time

[[citation needed](#)] ;p

But you are right about the phrasing